settings.gradle.kts

```diff
@@ -79,8 +82,10 @@ dependencyResolutionManagement {
             bundle("bouncycastle", listOf("bouncycastle-bcprov-jdk15on", "bouncycastle-bcpkix-jdk15on"))
             // Use when you need to make use of dependency injection.
             bundle("di", listOf("javax-inject", "dagger-api"))
+            // Use when you need a grpc server
+            bundle("helidon", listOf("helidon-server", "helidon-grpc", "helidon-io-grpc"))
             // Use when you need logging
-            bundle("logging", listOf("log4j-api", "log4j-core"))
+            bundle("logging", listOf("log4j-api", "log4j-core", "log4j-slf4j", "slf4j-api"))
```

Comment:
I would not define a logging bundle that contains all this dependencies since some should not be available at compiletime.

From my point of view we should only have a completive dependency to `slf4j-api` in all new modules. `log4j-slf4j` and `log4j-api` / `log4j-core` should be defined as runtime-only dependencies.

I assume we have the same problem with some other bundles. That's why I'm not a fan of bundles: You can not do the compile / runtime split.